execution/executor-types/src/in_memory_state_calculator.rs

```diff
@@ -81,7 +80,7 @@ impl InMemoryStateCalculator {
             latest: current.freeze(),
             next_version: current_version.map_or(0, |v| v + 1),
             updated_between_checkpoint_and_latest: updated_since_checkpoint,
-            updated_after_latest: HashSet::new(),
+            updates_after_latest: HashMap::new(),
```

Comment:
@msmouse any objection?

Reply:
I might be okay :trollface:  but I'm not too sure why you are changing it.

Reply:
Since I ready output the (k, v)s per version, it's better to cache them all as a map.

Reply:
correct me if I was wrong?